net/lighttpd/Makefile

```diff
@@ -156,6 +179,15 @@ $(eval $(call BuildPlugin,redirect,URL redirection,+PACKAGE_lighttpd-mod-redirec
 # Next, permit authentication.
 $(eval $(call BuildPlugin,auth,Authentication,,20))
 $(eval $(call BuildPlugin,authn_file,File-based authentication,,20))
+ifneq (,$(CONFIG_PACKAGE_lighttpd-mod-authn-gssapi))
```

Comment:
I do not think this is right. I did not see the option to select this build feature in "make menuconfig" until I removed the ifneq/endif surrounding each of these $(eval $(call BuildPlugin...)) statements (authn_gssapi, authn_ldap, etc.).